packages/api/src/controllers/room.ts

```diff
@@ -64,14 +72,163 @@ function toExternalRoom(room: Room, isAdmin = false) {
   if (isAdmin) {
     return room;
   }
-  room.userId = undefined;
-  for (const key in room.participants) {
-    room.participants[key].tracksPublished = undefined;
+  const roomForResp = {
+    id: room.id,
+    updatedAt: room.updatedAt,
+    createdAt: room.createdAt,
+    participants: room.participants,
+  };
```

Comment:
Are this all keys in the object? If so, this helper syntax might be your friend
```suggestion
  const roomForResp = { ...room };
```

If you are omitting some fields then nevermind me.

Reply:
We're omitting internal stuff like userId.